templates/shared_services/firewall/porter.yaml

```diff
@@ -45,7 +45,7 @@ parameters:
     type: string
     default: "W10=" # b64 for []
     description: "Network rule collection array"
-  - name: sku_tier
+  - name: firewall_sku
```

Comment:
I can't leave this comment in the right place since it's too far from the first change in the file, but I think you should also update this bundle's version on line 3.

Reply:
Thanks - added in https://github.com/nwsde/nwsde-azuretre/commit/ad5ac13907869df2efb262dd0a6b993b7df2b504